bedrock/foundation/templates/foundation/trademarks/distribution-policy.html

```diff
@@ -22,39 +22,39 @@
   </blockquote>
 
   <p>
-    {% trans %}
-      If you wish to distribute Firefox branded with our trademarks, you must comply with the Mozilla Public License, the Mozilla Trademark Guidelines and either the below Unaltered Firefox Distribution Policy or the terms of a written distribution agreement with Mozilla.
+    {% trans trademark_guidlines="{{ url('foundation.trademarks.policy.#firefox-add-on') }}" %}
```

Comment:
This link doesn't quite work as-is. Try:
```suggestion
    {% trans trademark_guidlines=url('foundation.trademarks.policy') + '#firefox-add-on' %}
```

Reply:
Also, is the Firefox Add-on section of this page the right target destination, given the copy?

Reply:
Fixed the link, the URL was expressly called out in the doc but I will check to make sure.

Reply:
Updated link